src/sass/app_2/modules/trading.scss

```diff
@@ -407,26 +420,77 @@
         }
     }
     &:disabled {
-        background-color: $COLOR_DARK_BLUE_8;
-        color: $COLOR_DARK_BLUE_7;
         @include themify($themes) {
-            background-color: themed('disabled_color');
-            color: themed('text_secondary_color');
+            background-color: themed('purchase_background_color');
+        }
+        color: $COLOR_DARK_BLUE_7;
+
+        & span {
+            display: block;
+        }
+        .btn-purchase__content {
+            @include themify($themes) {
+                background-color: themed('purchase_detail_background_color');
+            }
+        }
+        .btn-purchase__effect-detail {
+            @include themify($themes) {
+                border-color: transparent transparent transparent themed('purchase_detail_background_color');
+            }
+        }
+        .btn-purchase__trade-type-icon, .btn-purchase__trade-type-text, .btn-purchase__profit, .btn-purchase__return {
```

Comment:
Is there a way we can unify all these classes that share this specific rule instead of having to reference all of the classes in one line? Same thing applies for L444, L448, L453 and L457.

Reply:
I agree, they looks super ugly

Reply:
will it look better if we can seperate the comma to new line instead of space ?

Reply:
@cakasuma , I don't think that will pass our `stylelint` rule-set.

Reply:
@cakasuma Yea, it doesn't pass it.

Reply:
I have unified the classes into 1!